src/styles/default/pretty.jl

```diff
@@ -320,7 +319,7 @@ function format_docstring(style::AbstractStyle, state::State, text::AbstractStri
                     write(deindented, line)
                 else
                     write(deindented, '\n')
-                    write(deindented, SubString(line, user_indent + 1, length(line)))
```

Comment:
Similarly, here generally neither `user_indent + 1` nor `length(line)` (which is the number of characters, not necessarily the last index!) might be a valid index.